lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -3147,7 +3206,13 @@ class MongoClientInterface {
     deleteBucketIndexes(bucketName: string, indexSpecs: { name: string }[], log: werelogs.Logger, cb: ArsenalCallback<void>) {
         const c = this.getCollection<ObjectMetastoreDocument>(bucketName);
         async.each(indexSpecs,
-            (spec, next) => c.dropIndex(spec.name).then(() => next()).catch(err => next(err)),
+            (spec, next) => c.dropIndex(spec.name).then(result => {
```

Comment:
probably OK if the index is already deleted : could be handled here, or return an error `NoSuchIndex` or `DeleteConflict` so caller can manage...